chainer/functions/evaluation/classification_summary.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import division
+
+import numpy
+import six
+
+import chainer
+from chainer import cuda
+from chainer import function
+from chainer.utils import type_check
+
+
+def _f1_score(precision, recall, beta):
+    beta_square = beta * beta
+    return ((1 + beta_square) * precision * recall /
+            (beta_square * precision + recall)).astype(precision.dtype)
+
+
+class ClassificationSummary(function.Function):
+
+    ignore_label = -1
+
+    def __init__(self, beta, label_num):
```

Comment:
Thank you for the suggestion. It is just because I could not spare my time to consider the test cases (especially, the corner cases). Further, as we can expand the API afterwards, I sent this PR without `ignore_label` option. So I do not implement the option somewhat intentionally. But as you may think, it is better, to provide users with the interface that changes `ignore_label` easily. So I will consider adding it.

By the way, `F.accuracy` can change the value used as `ignore_label`, whereas it is fixed to `-1` in `F.binary_accuracy`, `F.sigmoid_cross_entropy` and `F.softmax_cross_entropy`. So do you think we should admit users to flexibly change the value of `ignore_label`?

But as you may think, it is better to add `ignore_label`